cli/src/commands/auth.rs

```diff
@@ -29,30 +41,46 @@ async fn handle_auth_login(
 ) -> Result<()> {
     let api_uri = &config.connection.uri;
     let ignore_certs = config.ignore_certs();
-    config.auth_info =
-        PhylumApi::login(config.auth_info, ignore_certs, api_uri, matches.get_flag("reauth"))
-            .await?;
+    config.auth_info = PhylumApi::login(
+        config.auth_info,
+        matches.get_one("token-name").cloned(),
+        ignore_certs,
+        api_uri,
+        matches.get_flag("reauth"),
+    )
+    .await?;
     save_config(config_path, &config).map_err(|error| anyhow!(error))?;
     Ok(())
 }
 
 /// Display the current authentication status to the user.
 pub async fn handle_auth_status(config: Config, timeout: Option<u64>) -> CommandResult {
-    if config.auth_info.offline_access().is_none() {
-        print_user_warning!("User is not currently authenticated");
-        return Ok(ExitCode::NotAuthenticated);
-    }
+    let auth_type = match config.auth_info.offline_access() {
+        Some(token) if is_locksmith_token(token) => "API key",
+        Some(_) => "OpenID Connect",
+        None => {
+            print_user_warning!("User is not currently authenticated");
+            return Ok(ExitCode::NotAuthenticated);
+        },
+    };
 
     // Create a client with our auth token attached.
     let api = PhylumApi::new(config, timeout).await?;
 
     let user_info = api.user_info().await;
 
+    debug!("User info reponse: {:?}", user_info);
```

Comment:
Just making sure this new log is intentional and not left over from debugging.

Reply:
Yes. It's intentional